index.js

```diff
@@ -188,37 +130,50 @@ function makeCachedResolver (prefix, fieldName, cache, originalFieldResolver, po
       // We must skip ctx and info as they are not easy to serialize
       return { self, arg, fields, extendKey }
     }
-  }, async function ({ self, arg, ctx, info, extendKey }, key) {
-    if (storage) {
-      const val = await storage.get(name + '~' + key)
-      if (val) {
-        onHit()
-        return val
-      }
-    }
-
-    onCacheMiss ? onCacheMiss() : onMiss()
-    const res = await originalFieldResolver(self, arg, ctx, info)
-    if (storage) {
-      await storage.set(name + '~' + key, res)
-    }
-    return res
+  }, async function ({ self, arg, ctx, info }) {
+    return originalFieldResolver(self, arg, ctx, info)
   })
+
   return async function (self, arg, ctx, info) {
-    const isMutation =
-      info.operation && info.operation.operation === 'mutation'
-    if (
-      (skip && (await skip(self, arg, ctx, info))) ||
-      isMutation ||
-      (policy && policy.skip && (await policy.skip(self, arg, ctx, info)))
-    ) {
-      if (!isMutation) {
-        onCacheSkip ? onCacheSkip() : onSkip()
+    let result
+    try {
+      // dont use cache on mutation
+      // TODO dont cache also subscriptions
```

Comment:
should this TODO be solved?

Reply:
it's not related to the PR, just a note to open the issue later